client/src/components/NotFound.js

```diff
@@ -1,6 +1,6 @@
-import React, {Component} from 'react'
+import React, { PureComponent } from 'react'
 
-export default class NotFound extends Component {
+export default class NotFound extends PureComponent {
```

Comment:
Why the change from Component to PureComponent?

Reply:
Since it is a stateless component, this seemed more logical. Would prefer a functional component, but it messes with the global css state. (did not see a quick fix there)

Reply:
Then I would prefer to see it in its own separate commit with a clear message, since it bears no direct relation to the referenced issue.